src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+import { TabContent } from '../TabContent';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId = '' } = useParams();
+
+  const selectedTab = tabs.find(tab => tab.id === tabId);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              className={classNames({
+                'is-active': tabId === tab.id,
+              })}
+              key={tab.id}
+            >
+              <Link to={`/tabs/${tab.id}`}>{tab.title}</Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <TabContent tabContent={selectedTab?.content} />
```

Comment:
`
 const content
  = tabs.find(tab => tab.id === tabId)?.content || 'Please select a tab';
`

I made this like and I don`t need the separate component

Reply:
I'm not sure about it, theoretically content of the tab can be more than just one sentence and it seems to me that it's rather a separate component

Reply:
should we show `TabContent` component if there is no `selectedTab`?